exporter/opentelemetry-exporter-prometheus-remote-write/README.rst

```diff
@@ -1,27 +1,248 @@
-OpenTelemetry Prometheus Remote Write Exporter
-==============================================
+OpenTelemetry Python SDK Prometheus Remote Write Exporter
+=========================================================
 
-This library allows exporting metric data to `Prometheus Remote Write Integrated Backends
-<https://prometheus.io/docs/operating/integrations/>`_. Latest `types.proto
-<https://github.com/prometheus/prometheus/blob/master/prompb/types.proto>` and `remote.proto
-<https://github.com/prometheus/prometheus/blob/master/prompb/remote.proto>` Protocol Buffers 
-used to create WriteRequest objects were taken from Prometheus repository. Development is 
-currently in progress.
+This package contains an exporter to send `OTLP`_ metrics from the
+Python SDK directly to a Prometheus Remote Write integrated backend
```

Comment:
What is the "Python SDK"? We should be specific and also link to the github page where the OT SDK exists (or the pypi page).

Reply:
added!